cmd/adsysd/integration_tests/adsys_test.go

```diff
@@ -485,7 +485,7 @@ type runner interface {
 	Run() error
 }
 
-func TestExecuteCommand(t *testing.T) {
+func TestExecuteCommand(_ *testing.T) {
```

Comment:
shouldn’t we just change the signature?

Reply:
Maybe we should deactivate `revive` linting on test files?

Reply:
oh, you’re right it’s really a test. So yeah, I wouldn’t disable `revive`, because it’s useful to track unused variables, but having it explicitely ignored is fine then IMHO.